DNN Platform/DotNetNuke.Abstractions/Skins/ISkinPackageInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Skins;
+
+using System.Collections.Generic;
+
+/// <summary>The skin package info.</summary>
+public interface ISkinPackageInfo
+{
+    /// <summary>Gets or sets the ID of the package.</summary>
+    int PackageId { get; set; }
+
+    /// <summary>Gets or sets the ID of the skin package.</summary>
+    int SkinPackageId { get; set; }
+
+    /// <summary>Gets or sets the ID of the portal.</summary>
+    /// <remarks>If the portal ID is <c>-1</c>, then the skin package is a global skin package.</remarks>
+    int PortalId { get; set; }
+
+    /// <summary>Gets or sets the name of the skin.</summary>
+    string SkinName { get; set; }
+
+    /// <summary>Gets the skins in the skin package.</summary>
+    IList<ISkinInfo> Skins { get; }
```

Comment:
If we're going to use a collection interface, I think the read-only version is more expressive of the intent of the method (i.e. changing the collection would/should not have any effect).
```suggestion
    IReadOnlyList<ISkinInfo> Skins { get; }
```

Reply:
The problem however is that you can't add a new skin to the package, but you can edit an existing skin:

```cs
ISkinPackageInfo skinPackageInfo; // Instance of the package

ISkin newSkin = skinService.CreateSkin();
skinPackageInfo.Skins.Add(newSkin); // Error!

ISkin existingSkin = skinPackageInfo.Skins[0];
existingSkin.SkinSrc = "New.ascx"; // Changed... So are we read-only or not 🤔

skinService.UpdateSkinPackage(skinPackageInfo)
```